2_instance_localnet/deploy.tf

```
@@ -15,12 +15,27 @@ resource "openstack_networking_subnet_v2" "subnet" {
   dns_nameservers = ["1.1.1.1", "9.9.9.9"]
 }
 
+data "openstack_images_image_v2" "image" {
+  name = var.image_name
+  most_recent = true
```

This change is already merged, but here is an afterthought:

```suggestion
visibility = "public"
most_recent = true
```

If I read [the documentation](https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs/data-sources/images_image_v2) correctly, then the default value for `visibility` is `"private"`, and we probably want to restrict the image search/match to just public images instead.

---

Ah, yes, that makes sense. And in any case, it doesn't hurt to be explicit about visibility.